.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - node
+  - 16
```

Comment:
any reason for this change?

Reply:
yes

The installed version is borked. I have noticed it in another PR.